### ISSUE TITLE
Color highlighting for resource names

### ISSUE THREAD

**Issue by andyshinn** [russCloak/SublimePuppet#2]
When you define a resource like:

```puppet
file { '/etc/php.d/timezone.ini':
  ensure  => 'file',
  content => "date.timezone = ${timezone}"
}
```

The highlighting for the entire line is a solid color. But when you add a space after the `:` it looks much better:

```puppet
file { '/etc/php.d/timezone.ini' :
  ensure  => 'file',
  content => "date.timezone = ${timezone}"
}
```

It would be nice to have both styles display color highlighting correctly.

I presume it has something to do with the `meta.definition.resource.puppet` in the `SymbolList (resource).tmPreferences` file. I played with it a bit. But can't figure out how it is defining colors. Here are some screenshots to illustrate:

Good colors (good syntax):
![screen shot 2013-08-01 at 2 46 11 pm](https://f.cloud.github.com/assets/315485/897516/2aa2a230-faf4-11e2-8658-818f19fa7080.png)

Bad colors (good syntax):
![screen shot 2013-08-01 at 2 46 20 pm](https://f.cloud.github.com/assets/315485/897508/0bbc668a-faf4-11e2-88d2-c374bd378f04.png)

**Comment by petems** [comment]
@andyshinn I know this is a long time after you posted this (3 years! :sob:) but I think this is now fixed by #20 and #19 :

<img width="461" alt="screenshot 2016-02-15 11 15 08" src="https://cloud.githubusercontent.com/assets/1064715/13047200/64f2d7ae-d3d5-11e5-93b6-6763a93b1f98.png">

If youre still experiencing this issue, please say! Closing for now :+1:

**Comment by andyshinn** [comment]
Been a long while since I've used either of them. But thanks for the fix!
